fixtures/components/third-party-import-types/button/interfaces.ts

```diff
@@ -1,6 +1,6 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
-import { IconProps } from '../node_modules_mock/icon';
+import { IconProps } from 'icon';
```

Comment:
Updated this test to more accurately represent `node_modules` resolution. `node_modules_mock` is a normal project folder, typescript has no special logic there.